packages/gatsby-telemetry/src/__tests__/error-helpers.js

```diff
@@ -0,0 +1,90 @@
+const { sanitizeError, cleanPaths } = require(`../error-helpers`)
+
+describe(`Errors Helpers`, () => {
+  describe(`sanitizeError`, () => {
+    it(`Removes env, output and converts buffers to strings from execa output`, () => {
+      const tags = {
+        error: [
+          {
+            error: null,
+            cmd: `git commit -m"test"`,
+            file: `/bin/sh`,
+            args: [`/bin/sh`, `-c`, `git commit -m"test`],
+            options: {
+              cwd: `here`,
+              shell: true,
+              envPairs: [`VERSION=1.2.3`],
+              stdio: [{}, {}, {}], // pipes
+            },
+            envPairs: [`VERSION=1.2.3`],
+
+            stderr: Buffer.from(`this is a test`),
+            stdout: Buffer.from(`this is a test`),
+          },
+        ],
+      }
+
+      const error = tags.error[0]
+      expect(error).toBeDefined()
```

Comment:
i don't feel like you need these tests
I would just go for
```js
sanitizeError(tags)
expect(error.stderr).toStrictEqual(`this is a an error`)
expect(error.stdout).toStrictEqual(`this is a test`)
expect(error.envPairs).toBeUndefined()
expect(error.options).toBeUndefined()
```

Reply:
This makes a lot of sense! Done